resources/definitions/lulzbot_mini_2_se.def.json

```diff
@@ -0,0 +1,38 @@
+{
+   "version": 2,
+   "name": "Lulzbot Mini 2 | SE | 0.5 mm",
+   "inherits": "lulzbot_mini_2_common",
+   "metadata": {
+        "author": "Tom Callaway <spot@fedoraproject.org>",
+        "manufacturer": "Fargo Additive Manufacturing Equipment 3D, LLC",
+        "visible": true,
+        "file_formats": "text/x-gcode",
+        "platform": "lulzbot_mini.stl",
+        "platform_offset": [-100, -11, 100],
+        "machine_extruder_trains": {
+            "0": "lulzbot_mini_2_se_extruder"
+        }
+   },
+
+   "overrides": {
+       "machine_name": { "default_value": "Lulzbot Mini 2 | SE | 0.5 mm" },
+       "machine_width": { "default_value": 162 },
+       "machine_depth": { "default_value": 162 },
+       "machine_height": { "default_value": 180 },
```

Comment:
In the [specifications](https://shop.lulzbot.com/lulzbot-mini-v2-0-boxed-for-retail-na-kt-pr0047na) it says 160x160x180. Are the dimensions here correct?

Reply:
Those values were taken from https://gitlab.com/lulzbot3d/cura-le/cura-lulzbot/-/blob/master/resources/definitions/lulzbot_hibiscus.def.json. Not sure why there is a disconnect. This printer has a wipe pad so perhaps that explains the additional width.